lib/purl_parser.rb

```diff
@@ -1,12 +1,13 @@
 # class used to represent a purl, used to parse information
```

Comment:
I'm not sure... Its part of the standard library and should be required within a Rails application. I could go either way :) Happy to add it if you think it would be helpful